Makefile

```diff
@@ -28,13 +28,14 @@ local-build:
 ## Build the docker image
 docker:
 	@echo 'Creating docker image ${DOCKER_IMAGE}:${COMMIT}'
-	@docker buildx create --use --name=multiarch --node multiarch && \
+	docker buildx create --use --name=multiarch --driver-opt network=host --buildkitd-flags '--allow-insecure-entitlement network.host' --node multiarch && \
```

Comment:
Can you explain why this is needed? Does this gets baked into an image somehow?

Reply:
This is marking the container as requiring host networking; i.e., the container is (allowed to be) placed in the host's network namespace.  This is so the gRPC wires go direct from node to node rather than routed through the container networking layer.

Reply:
Yes @Cerebus this is precisely the reason. Host network is used as the back end for meshnet daemon set communicate with each other. No need to go though container networking layers.

Reply:
Thanks. I was not familiar with docker image entitlements, but have read up on them and now it makes sense.